scripts/strava_to_garmin_sync.py

```diff
@@ -73,31 +72,33 @@ def make_gpx_from_points(title, points_dict_list):
     return gpx.to_xml()
 
 
-async def upload_to_activities(garmin_client, strava_client):
+async def upload_to_activities(garmin_client, strava_client, strava_web_client, format):
     last_activity = await garmin_client.get_activities(0, 1)
     if not last_activity:
+        print("no garmin activity")
         filters = {}
     else:
         # is this startTimeGMT must have ?
         after_datetime_str = last_activity[0]["startTimeGMT"]
         after_datetime = datetime.strptime(after_datetime_str, "%Y-%m-%d %H:%M:%S")
+        print("garmin last activity date: ", after_datetime)
         filters = {"after": after_datetime}
     strava_activities = list(strava_client.get_activities(**filters))
     files_list = []
+    print("strava activities size: ", len(strava_activities))
+    if not strava_activities:
+        print("no strava activity")
+        return files_list
+
     # strava rate limit
-    for i in strava_activities[:50]:
-        start_time = i.start_date
-        activity_type = i.type
-        # strava steams types info from strava api doc
-        types = ["time", "latlng", "altitude", "heartrate", "velocity_smooth", "temp"]
-        s = strava_client.get_activity_streams(i.id, types=types, resolution="medium")
-        points = generate_strava_run_points(start_time, s)
-        if points:
-            garmin_type = STRAVA_GARMIN_TYPE_DICT.get(activity_type, "running")
-            gpx_doc = make_gpx_from_points("test", points)
-            file = BytesIO(bytes(gpx_doc, "utf8"))
-            files_list.append((file, garmin_type))
-    await garmin_client.upload_activities(files_list)
+    for i in strava_activities[:len(strava_activities)]:
+        try:
+            print(i.id)
```

Comment:
drop this print or add some info maybe better

Reply:
ok.Let me change that